cmds/exp/tcpdump/main.go

```diff
@@ -329,6 +329,8 @@ func (cmd *cmd) processPacket(packet gopacket.Packet, num int, lastPkgTimeStamp
 			data = fmt.Sprintf("UDP, length %d", length)
 		case *layers.UDPLite:
 			data = fmt.Sprintf("UDPLite, length %d", length)
+		case nil:
+			data = fmt.Sprintf("unknown transport layer, length %d", length)
```

Comment:
nit (not part of your change): you actually can remove else part from this chunk:

```go
		var length int
		if applicationLayer != nil {
			length = len(applicationLayer.LayerContents())
		} else {
			length = 0
		}
```
var length int will be zero anyway

Reply:
you don't need the else do you? length is 0 already.

Reply:
ah I misread binjip978 comment, I am agreeing with him. please remove that else.

Reply:
Thats right! I pushed another commit. :+1: